### ISSUE TITLE
FURB105 should detect empty f-strings

### ISSUE THREAD

**Issue by dscorbett** [astral-sh/ruff#21346]
### Summary

It would be helpful for [`print-empty-string` (FURB105)](https://docs.astral.sh/ruff/rules/print-empty-string/) to detect empty f-strings via [`is_empty_f_string`](https://github.com/astral-sh/ruff/blob/0.14.4/crates/ruff_python_ast/src/helpers.rs#L1414). [Example](https://play.ruff.rs/210c3a04-8fb6-43d3-a39c-4caed9335c1d):
```console
$ cat >furb105.py <<'# EOF'
print(f"")
# EOF

$ ruff --isolated check furb105.py --select FURB105
All checks passed!
```

### Version

ruff 0.14.4 (c7ff9826d 2025-11-06)
